cmd/mesh/profile-list.go

```diff
@@ -0,0 +1,48 @@
+// Copyright 2019 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package mesh
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+)
+
+type profileListArgs struct {
+	// set is a string with element format "path=value" where path is an IstioControlPlane path and the value is a
+	// value to set the node at that path to.
+	set string
```

Comment:
This command is not yet implemented, just curious where will this flag be used? From the description, it looks like it would set the node at specific path with given value, then why would it be part of "list" command?I thought list command here is doing something immutable

Reply:
Agreed that it is surprising to see a mutation flag on a list operation which is typically read-only.

Reply:
This was a copy & past error - fixed.